example/example.c

```diff
@@ -44,7 +45,13 @@ int report_error(void);
  * @returns EXIT_SUCCESS if execution completes successfully, or EXIT_FAILURE
  * if an error occurs.
  */
-int main(void) {
+int main(int argc, char** argv) {
+#if defined(__HAIKU__) && defined(NDEBUG)
+    disable_debugger(1);
```

Comment:
what does disaable_debugger do, besides disable the debugger?

Reply:
That's it.  If you don't call that, then by default a graphical debugger interface is attached to any crashing program.  It's nice if you are using the system, but if you connect with SSH or running via a script, the process will just appear to hang forever, no visible crash or core file or anything.